tests/roles/keystone_adoption/tasks/main.yaml

```diff
@@ -87,3 +87,9 @@
     for service in aodh heat heat-cfn barbican cinderv3 glance manila manilav2 neutron nova placement swift; do
       ${BASH_ALIASES[openstack]} service list | awk "/ $service /{ print \$2; }" | xargs ${BASH_ALIASES[openstack]} service delete || true
     done
+
+    {% if ironic_adoption | default(false) %}
+    for service in ironic-inspector ironic; do
+      ${BASH_ALIASES[openstack]} service list | awk "/ $service /{ print \$2; }" | xargs ${BASH_ALIASES[openstack]} service delete || true
+    done
+    {% endif %}
```

Comment:
IMO let's just use the same thing that's in the docs? Thanks to the `|| true` at the end, it won't fail if the service isn't found. I think the Ceph scenario doesn't have Swift deployed either.